packages/manager/modules/pci/src/projects/project/sidebar/project-list/project-list.html

```diff
@@ -19,8 +19,16 @@
         </div>
         <div class="mt-3" data-ng-if="$ctrl.projects && $ctrl.projects.length > 0">
             <strong class="text-white" data-translate="cloud_sidebar_projectlist_title"></strong>
+            <div class="oui-search mt-2">
```

Comment:
Why not using directly the [`<oui-search>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/search) component?

Reply:
The `oui-search` doesn't correspond to the CX spec I got